docs/functional-components/static-assets.md

```diff
@@ -0,0 +1,26 @@
+## Static assets
+
+## Recommended packages
+
+* [express-static](https://expressjs.com/en/starter/static-files.html). 
+Express static is part of the express.js package that allows developers to expose static middlewares.
+
+## Guidance
+
+For serving static resources we recomend using `express.static` middleware as it has been widely used and tested in production.
+When using express.static() method, we can server static resources directly by specifying the folder name where we have stored our static resources.
+
+Documentation for the middleware can be found in:
+https://expressjs.com/en/starter/static-files.html
+
+### Caching
+
+The static middleware does server-side caching. 
+It lets you do two methods of client-side caching: ETag and Max-Age
+If you configured your static middleware to use client side caching please make sure that 
```

Comment:
Mention that ETag is not required. CacheControl only. 
Disable ETAG.

Reply:
https://stackoverflow.com/a/58122775/12587417

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag

Reply:
https://www.freecodecamp.org/news/an-in-depth-introduction-to-http-caching-cache-control-vary/ reference for images and decision/sequence diagrams